{"name": "test_login_fail[fail_data0]", "status": "passed", "description": "\n        用例描述：测试登录失败用例\n        ", "attachments": [{"name": "log", "source": "52beb86f-33e5-4dec-a7d2-a7f35898891e-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "23240f1b-8362-4c5e-ba20-d09330eee226-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "fail_data", "value": "{'sUserName': '16533907261', 'sPassword': 'xunyuji123'}"}], "start": 1587118960319, "stop": 1587118961022, "uuid": "0e8a6369-21c4-4de6-9e95-968466529ebb", "historyId": "32ce2a043e6f3b1af9f51845c6c9e382", "testCaseId": "8000cb86548fa2cebd0d986e95a5b795", "fullName": "test_case.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "登录失败用例"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admin"}, {"name": "thread", "value": "11288-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}